diagnostic.md

```diff
@@ -5,20 +5,26 @@ Record your responses inside the fenced code blocks below each question.
 1.  Give an example of a visual hierarchy that could be modeled with components.
 
     ```md
-    <!-- your response here -->
+    A tree can be a visual representation of components, this can show how
+    components can be nested within other components, and how they relate to
+    outer routes, etc.
     ```
 
 1.  What is the command to generate a new component called '`my-map`'?
 
     ```sh
-    # your response here
+    ember g component my-map
     ```
 
 1.  What files are edited to produce a component, and what are their
     responsibilities?
 
     ```md
-    <!-- your response here -->
+    A component.js, template.hbs, and component tests are created when a
+    component is generated. The template file is the representation of the
+    component on the web page. The component file is where you can place actions
+    or tagnames that relate to the template file. Lastly, the component tests
+    are made by ember for testing.
```

Comment:
Looks good.  The `template.hbs` is also responsible for binding data, and binding and dispatching actions.